packages/synapse-react-client/src/components/widgets/query-filter/CombinedRangeFacetFilter.tsx

```diff
@@ -0,0 +1,157 @@
+import { Collapse } from '@mui/material'
```

Comment:
Core of the change.  Originally tried to change RangeFacetFilter to support both cases, but this is so different that I'd like to isolate the logic.